### PR TITLE
chore(css): Remove Box/Flex from choiceMapperField

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/components/forms/choiceMapperField.jsx [hunk 1]
```diff
@@ -230,10 +224,46 @@ export default class ChoiceMapper extends React.Component {
...
 
-const LabelColumn = styled(p => <Box flex="0 0 200px" {...p} />)``;
+const Header = styled('div')`
+  display: flex;
+  align-items: center;
+`;
+
+const Heading = styled('div')`
+  display: flex;
+  margin-left: ${space(1)};
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/views/settings/components/forms/choiceMapperField.jsx]: Please, are you sure about this margin? `space(1)` returns` 8px` and before this margin was` 1px`

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/settings/components/forms/choiceMapperField.jsx]: Reflex box's `m*` and `p*` attributes used a similar spacing config

Comment (priscilawebdev) [reply src/sentry/static/sentry/app/views/settings/components/forms/choiceMapperField.jsx]: Yeah I didn't know... Billy told me the same yesterday... thanks :)

**Review by priscilawebdev — APPROVED** [review]
LGTM ... good job 👍

**Pull request merged**